openassessment/templates/openassessmentblock/edit/oa_edit.html

```diff
@@ -101,15 +101,32 @@ <h2 class="openassessment_alert_title">{% trans "Rubric Change Impacts Settings
+                <li id="openassessment_submission_file_wrapper" class="field comp-setting-entry">
                     <div class="wrapper-comp-setting">
-                        <label for="openassessment_submission_image_editor" class="setting-label">{% trans "Allow Image Responses"%}</label>
-                        <select id="openassessment_submission_image_editor" class="input setting-input" name="image submission">
-                            <option value="0">{% trans "False"%}</option>
-                            <option value="1" {% if allow_file_upload %} selected="true" {% endif %}>{% trans "True"%}</option>
+                        <label for="openassessment_submission_upload_selector" class="setting-label">{% trans "Allow File Upload"%}</label>
+                        <select id="openassessment_submission_upload_selector" class="input setting-input" name="upload submission">
+                            <option value="none">{% trans "None"%}</option>
+                            <option value="image" {% if file_upload_type == "image" %} selected="true" {% endif %}>{% trans "Image File"%}</option>
+                            <option value="file" {% if file_upload_type == "file" %} selected="true" {% endif %}>{% trans "PDF or Image File"%}</option>
+                            <option value="custom" {% if file_upload_type == "custom" %} selected="true" {% endif %}>{% trans "Custom File Types"%}</option>
                         </select>
                     </div>
-                    <p class="setting-help">{% trans "Specify whether students can submit an image file along with their text response." %}</p>
+                    <p class="setting-help">
+                        {% trans "Specify whether students can submit a file along with their text response. 'Image' option allows JPG, GIF and PNG files. 'PDF or Image' option allows PDF and images. The 'Custom File Types' option allowes files with white listed extensions to be uploaded (use this option with caution)." %}
+                    </p>
+                    <div id="openassessment_submission_file_type_white_list_wrapper" class="{% if file_upload_type != "custom" %}is--hidden{% endif %}">
+                        <div class="wrapper-comp-setting">
+                            <label for="openassessment_submission_file_type_white_list" class="setting-label">{% trans "White List" %}</label>
+                            <input id="openassessment_submission_file_type_white_list"
+                                   class="input setting-input"
+                                   type="text"
+                                   value="{{ file_type_white_list }}"
+                                    />
+                        </div>
+                        <p class="setting-help">
+                            {% trans "Comma separated file extensions that are allowed to be uploaded. For example: 'pdf,doc,docx'" %}
+                        </p>
```

Comment:
I'd suggest,
Enter file extensions, separated by commas, that you want students to be able to upload. For example: pdf,doc,docx